includes/definitions/fiberhome-switch.yaml

```diff
@@ -0,0 +1,19 @@
+os: fiberhome-switch
+text: 'FiberHome Switch'
+type: network
+icon: fiberhome
+discovery:
+    - sysObjectID:
+        - .1.3.6.1.4.1.3807.1.582815
+        - .1.3.6.1.4.1.3807.1.281802
+        - .1.3.6.1.4.1.3807.1.481005
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+mib_dir:
+    - fiberhome
+discovery_modules:
```

Comment:
All of these should be default so just remove the discovery and poller modules.

Reply:
noted and done.